CHANGELOG.md

```diff
@@ -4,14 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+- Allow omitting `id`, defaulting to plugin type if unique within namespace
+- Allow omitting `output`, defaulting to the next operator in the pipeline if valid
```

Comment:
Can you give a case where this doesn't work as expected? The plugin is rendered directly into the array of plugins, so my understanding is that it works this way naturally

Reply:
I may have misinterpreted the meaning of this: `When an operator is the last in the pipeline, an error is thrown`. My assumption was that this referred to the last operator in any pipeline (plugin or otherwise).

Some of the new examples explicitly show using `{{ .output }}`, such as `examples/simple_plugins/plugins/decorator.yaml`. Is it optional in this case?

Reply:
They mean slightly different things. If `{{.output}}` is omitted, it will always output to the next operator in the pipeline (after we've replaced the plugin with the rendered operators) since we're ignoring the "output" field, which is why I left it in the example.

Reply:
Ok, I understand now that it is optional. Thanks for clarifying.